assets/src/edit-story/components/dropTargets/karma/background.karma.js

```diff
@@ -367,6 +363,121 @@ describe('Background Drop-Target integration', () => {
           expect(combinedBgTransform2).toBe('matrix(-1, 0, 0, 1, 0, 0)');
         });
       });
+
+      describe('when there is a third flipped image on the canvas', () => {
```

Comment:
Hm, shouldn't these belong to a separate file? It's a bit confusing that something that has nothing to do with background is under the background Karma. Maybe rename the file if it's mainly for drop-targeting and not for background specifically?

Reply:
I know, I was a bit conflicted about this too. I'll rename the test file to better reflect the test suites contained therein.